lib/matplotlib/tests/README.rst

```diff
@@ -4,5 +4,5 @@ About Matplotlib Testing Infrastructure
 Information on the testing infrastructure is provided in
 the Testing section of the Matplotlib Developers’ Guide:
 
-* https://matplotlib.org/devel/testing.html
+* [Website](https://matplotlib.org/stable/devel/testing.html)
 * <mpl_source_dir>/doc/devel/coding_guide.rst (equivalent, but in reST format)
```

Comment:
Should probably point at https://github.com/matplotlib/matplotlib/blob/main/doc/devel/testing.rst